messages_api/views.py

```diff
@@ -7,18 +7,20 @@
 class MessageApiView(RetrieveAPIView):
     serializer_class = MessageSerializer
     text = None
+    api = "api_django_python_hello-world"
```

Comment:
I think it's better to have this as a default value inside the `Message` class because it's never going to change. Then the constructor can simply be `Message(text, branch)`

Reply:
Thinking about it. The `branch` parameter is not going to change either in during this whole branch. So we can move that one to the `Message` class too 🙂

Reply:
:+1: